cypress/utils/plugins/observability-dashboards/constants.js

```diff
@@ -80,7 +80,9 @@ export const setTimeFilter = (setEndTime = false, refresh = true) => {
       timeout: TIMEOUT_DELAY,
     }).type('{selectall}' + endTime, { force: true });
   }
-  if (refresh) cy.get('.euiButton__text').contains('Refresh').click();
+  if (refresh) {
+    cy.get('button[data-test-subj="superDatePickerApplyTimeButton"]').click();
```

Comment:
Same here. If you have a test subject, just use that. Tests will be les prone to error when code gets refactored

Reply:
done